components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/action/PreIssueAccessTokenResponseProcessor.java

```diff
@@ -743,4 +794,17 @@ private boolean isValidStringOrURI(String input) {
         Matcher matcher = STRING_OR_URI_PATTERN.matcher(input);
         return matcher.matches();
     }
+
+    private enum ServerErrorCode {
+        SERVER_ERROR("server_error"),
+        INTERNAL_SERVER_ERROR("internal_server_error"),
+        ERROR_500("500");
+
+        private final String code;
+
+        ServerErrorCode(String code) {
+
+            this.code = code;
+        }
+    }
```

Comment:
IMO, this is not necessary. why can't these be defined as private constants or directly code if used only in one place